modular_skyrat/master_files/code/controllers/subsystem/dbcore.dm

```diff
@@ -38,5 +38,5 @@
 	if(length(queued_log_entries_by_table[table]) < CONFIG_GET(number/sql_game_log_min_bundle_size))
 		return
 
-	MassInsert(table, rows = queued_log_entries_by_table[table], duplicate_key = FALSE, ignore_errors = FALSE, delayed = FALSE, warn = FALSE, async = TRUE, special_columns = null)
+	INVOKE_ASYNC(src, .proc/MassInsert, table, /*rows =*/ queued_log_entries_by_table[table], /*duplicate_key =*/ FALSE, /*ignore_errors =*/ FALSE, /*delayed =*/ FALSE, /*warn =*/ FALSE, /*async =*/ TRUE, /*special_columns =*/ null)
```

Comment:
why comment it all out?

Reply:
Because you can't actually use the variable names there, since it's a define and not a proc. I've left them there so it's easier to figure out which variable they represent, for the sake of the future, if a new variable gets added for whatever reason.